dataSource/src/main/java/com/panhuk/datasource/DatastorePreferences.kt

```diff
@@ -0,0 +1,8 @@
+package com.panhuk.datasource
+
+import kotlinx.coroutines.flow.Flow
+
+interface DatastorePreferences {
+   suspend fun saveSessionToken(token: String)
+   fun getSessionToken(): Flow<String?>
```

Comment:
Would be better to name it observeSessionToken() to emphasize that method returns flow of values

Reply:
+